src/control-manager.cpp

```diff
@@ -112,15 +111,107 @@ namespace dynamicgraph
+						    "(double) max current >0")));
+
+	addCommand("getDefaultMaxCurrent",
+		   makeDirectGetter(*this,&m_maxCurrent,
+				    docCommandVoid0("Get the default max current")));
+
+	addCommand("setNameToId",
+		   makeCommandVoid2(*this,&ControlManager::setNameToId,
+				    docCommandVoid2("Set map for a name to an Id",
+						    "(string) joint name",
+						    "(double) joint id")));
+
+	addCommand("setForceNameToForceId",
+		   makeCommandVoid2(*this,&ControlManager::setForceNameToForceId,
+				    docCommandVoid2("Set map from a force sensor name to a force sensor Id",
+						    "(string) force sensor name",
+						    "(double) force sensor id")));
+
+
+	addCommand("setJointLimitsFromId",
+		   makeCommandVoid3(*this,&ControlManager::setJointLimitsFromId,
+				    docCommandVoid3("Set the joint limits for a given joint ID",
+						    "(double) joint id",
+						    "(double) lower limit",
+						    "(double) uppper limit")));
+
+	addCommand("setForceLimitsFromId",
+		   makeCommandVoid3(*this,&ControlManager::setForceLimitsFromId,
+				    docCommandVoid3("Set the force limits for a given force sensor ID",
+						    "(double) force sensor id",
+						    "(double) lower limit",
+						    "(double) uppper limit")));
+
+	addCommand("setJointsUrdfToSot",
+		   makeCommandVoid1(*this, &ControlManager::setJoints,
+                                    docCommandVoid1("Map Joints From URDF to SoT.",
+                                                    "Vector of integer for mapping")));
+	
+	addCommand("setRightFootSoleXYZ",
+		   makeCommandVoid1(*this, &ControlManager::setRightFootSoleXYZ,
+                                    docCommandVoid1("Set the right foot sole XYZ.",
+                                                    "Vector of double")));
+
+	addCommand("setFootFrameName",
+                   makeCommandVoid2(*this, &ControlManager::setFootFrameName,
+                                    docCommandVoid2("Set the Frame Name for the Foot Name.",
+                                                    "(string) Foot name",
+                                                    "(string) Frame name")));
+		   
+        addCommand("displayRobotUtil",
+                   makeCommandVoid0(*this, &ControlManager::displayRobotUtil,
+                                    docCommandVoid0("Display the current robot util data set.")));
+
       }
 
-      void ControlManager::init(const double& dt)
+      void ControlManager::init(const double& dt, 
+				const std::string &urdfFile,
+				const double & lmax_current)
       {
         if(dt<=0.0)
           return SEND_MSG("Timestep must be positive", MSG_TYPE_ERROR);
+
+	m_maxCurrent = lmax_current;
+
         m_dt = dt;
         m_emergency_stop_triggered = false; 
         m_initSucceeded = true;
+	vector<string> package_dirs;
+
+	m_robot = new pininvdyn::RobotWrapper(urdfFile, package_dirs, se3::JointModelFreeFlyer());
+	
+	std::string localName("control-manager-robot");
```

Comment:
I know I already said it, but just as a reminder, this string "control-manager-robot" should be taken as an input parameter of the init command.